packages/fx-core/src/plugins/resource/bot/appStudio/appStudio.ts

```diff
@@ -114,11 +114,7 @@ export class AppStudio {
     }
 
     const app = response.data as IAADDefinition;
-    if (!app || !app.id || !app.appId) {
-      return false;
-    }
-
-    return true;
+    return !(!app || !app.id || !app.appId);
```

Comment:
I am not sure if it can be as below?
```
return app && app.id && app.appId;
```

Reply:
use this will case
TS2322: Type 'string | undefined' is not assignable to type 'boolean'.   Type 'undefined' is not assignable to type 'boolean'.